### ISSUE TITLE
Unable to instantiate decoder OMX.Exynos.avc.dec

### ISSUE THREAD

**Issue by AlexandruVoica** [TheWidlarzGroup/react-native-video#1255]
### Current behavior
I have 2 players at the same time. Only one of them works, the other throws an error with the message "Unable to instantiate decoder OMX.Exynos.avc.dec". One video file is .mov, the other one is .mp4

Anyone have any idea where to start debugging? 

### Expected behavior
Both players should work independently.

### Platform
Which player are you experiencing the problem on:
* Android ExoPlayer
* Samsung SM-A510F Android 7.0

**Comment by cobarx** [comment]
Have you tried the MOV file by itself? To the best of my knowledge ExoPlayer does not support MOV.

If it's not that, the video decoder written by Samsung for that device doesn't support multiple videos at the same time and there's nothing we can do.

**Comment by AlexandruVoica** [comment]
@cobarx thank you for you reply.

Is there any way to find more info about the decoder that is used by a particular device? I've scoured the internet and still have not found anything.

Since I'm very new to mobile development, I was unaware that Android does not support .mov files, which is the reason the decoder was failing. But, I've also tried with 3 .mp4 files and I get the same error when trying to load the 3rd video, so you must be right about the decoder not being able to support multiple videos at the same time, which is tedious to work around, to say the least.

I'm going to close this issue because it is apparent it is not related to react-native-video, but to the device specifically.
